### PR TITLE
Add notice that theta needs to be calculated after disp-regression. 

### PR DESCRIPTION
**Pull request by jebuss**
* Add a hint to the README  that theta needs to be calculated after disp-regression. 
* Add a hint to the command line in script `klaas.scripts.apply_disp_regressor.py` that theta needs to be calculated after disp-regression.

### CODE REVIEW

**Comment by jebuss** [comment]
fix #42

**Comment by mackaiver** [comment]
Can't we just calculate theta directly in here?

**Comment by MaxNoe** [comment]
Why this warning?
If not used with the `--yes` option, the script asks for confirmation:

https://github.com/fact-project/classifier-tools/blob/1b9030efd34c393b320e0b9b7a9bf6dd1bf2cec5/klaas/scripts/apply_disp_regressor.py#L61-L70

**Comment by jebuss** [comment]
> Why this warning?

My intention was to make it clear to the user that he deleted `theta` and how he can calculate it from the source prediction. Sure cmdline warnings are not the best instrument for this. 

However the description of the `yes` argument "'Do not prompt for overwrites'" did not make it  obvious to me that theta would be gone. Eventhough, now that I know, it is more obvious. I thus suggest to make this a bit more fool proof ;-)

> Can't we just calculate theta directly in here?

Since we don't have a variety of methods to swap, I think this would be use full. Even if you want to replace this calculation afterwards, you could simply overwrite theta again in a different program.

**Comment by MaxNoe** [comment]
As discussed in #42, I want to keep the disp regression source independent

**Comment by jebuss** [comment]
> As discussed in #42, I want to keep the disp regression source independent

Does this contradict with what Kai suggested? (If, how?)

**Comment by MaxNoe** [comment]
I though 

> directly in here

referred to `apply_disp_model`

**Comment by jebuss** [comment]
Ahhh, I see,
You mean if we do it here, you would calculate theta for a given source e.g. Crab along with the disp regression, but maybe you also want to calculate theta for a second source in the FOV.

Yes, I agree. It makes sense to separate these tasks. 

However, Since `theta` was there before and might be gone after, I think it makes sense to inform the user and also guide him to a tool that helps him*her.

**Comment by MaxNoe** [comment]
Yes, but I would move the warning up to the confirm call and maybe list all columns that got removed.
something like `logging.warning("Removing columns {}".format(col))`

File: klaas/scripts/apply_disp_regressor.py [hunk 1]
```diff
@@ -68,6 +68,11 @@ def main(configuration_path, data_path, disp_model_path, sign_model_path, key, c
                     )
                     yes = True
                 del f[key][column]
+                
+    log.warn("Deleted {} from the feature set.".format(columns_to_delete))
```
Comment (MaxNoe) [inline klaas/scripts/apply_disp_regressor.py]: I would only warn about the features that actually got removed. so move this up into the loop beneath the `del f[key[[column]` statement and log the removed feature.

**Pull request merged**